helsingborg-stad/app-mitt-helsingborg
# fix: remove Roboto font and run react-native link command

## Explain the changes you’ve made
Removed Roboto font-family from the application. 
Also did run react-native link command to link dependency which added new assets (fonts) for android

By removing the font, it also removes the error message received when starting the application saying that the Roboto font could not be found (FINALLY!)

## Explain why these changes are made
The Roboto font was not added to iOs properly, which meant that the iOS build did not contain any Roboto font, which means that the application would default to its system font instead.

Robot is also the default font in Android, which makes it even more unnecessary to add the font in the project.

## Explain your solution
Did remove all the font assets in the project

## How to test
1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Check if the font seems OK  

## Tested environments

- [] Storybook on a iOS device/simulator.

**Android**
![image](https://user-images.githubusercontent.com/81250970/145389923-2bf4002b-90e5-4ea0-8cd7-5204ca8f1c9d.png)

**iPhone**
![image](https://user-images.githubusercontent.com/81250970/145389967-55f6aeb2-686f-4687-9cea-3e40d1996cc8.png)
